<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="guessWinLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Curlz MT</family>
      <pointsize>72</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GuessWin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-GuessWin-Desktop_Qt_6_7_0_MinGW_64_bit-Debug/Files/Background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>271</width>
      <height>109</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="2">
      <widget class="QLineEdit" name="rateLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="userNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="userNameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>User Name: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="rateLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Rate: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="singleBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>391</y>
      <width>229</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Single Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiBtn">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>440</y>
      <width>229</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Multi Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rulesBtn">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>490</y>
      <width>229</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rules</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>guessWinLabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>backBtn</zorder>
   <zorder>singleBtn</zorder>
   <zorder>multiBtn</zorder>
   <zorder>rulesBtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionUser_Profile"/>
    <addaction name="actionLog_Out"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUser_Profile">
   <property name="text">
    <string>User Profile</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>multiBtn</tabstop>
  <tabstop>rulesBtn</tabstop>
  <tabstop>backBtn</tabstop>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>rateLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
